src/CMakeLists.txt

```diff
@@ -54,6 +60,13 @@ qt_add_qml_module(${PROJECT_NAME}
         QtQuick.Controls.Basic
 )
 
+list(APPEND QML_DIRS "/usr/lib64/qt6/qml")
+#TODO check QML output directory
+#list(APPEND QML_DIRS "${PROJECT_SOURCE_DIR}/src")
+list(REMOVE_DUPLICATES QML_DIRS)
+set(QML_IMPORT_PATH "${QML_DIRS}" CACHE STRING "Add QML imports" FORCE)
```

Comment:
Where is this variable used? Is it needed by QtCreator?

Reply:
I am terribly sorry, these needs to be moved in demo/CMakeLists.txt. I was trying to fix QML imports, going to revert these changes as they do not help with anything